src/romaine/core.py

```diff
@@ -14,6 +15,7 @@ def __init__(self):
             Initialise Romaine core.
         """
         self.steps = {}
+        self.parser = Parser()
```

Comment:
It is, but it feels like the core should have all of the components it used (well, feels like that to me anyway). Much the same as we have the ability to locate features in the core, this then gives the ability to do something with those features without leaving the core.

Happy to consider a different approach if we've got a consistent paradigm we're trying to stick to with the core.